src/components/Input/Input.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { v4 as uuid } from 'uuid';
+import './Input.scss';
```

Comment:
Are we planning on using CSS Modules to limit the scope of these styles? My concern is that we don't know what environment these components will end up in. With classnames like `.input` and `.label` it's very possible that these styles can bleed out.